src/acore-wp-plugin/src/Components/UnstuckMenu/UnstuckView.php

```
@@ -17,7 +19,7 @@ public function getUnstuckmenuRender($chars)
         ob_start();
 
         wp_enqueue_style('bootstrap-css', '//cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css', array(), '5.1.3');
-        wp_enqueue_style('acore-css', ACORE_URL_PLG . 'web/assets/css/main.css', array(), '0.1');
+        wp_enqueue_style('acore-css', ACORE_URL_PLG . 'web/assets/css/main.css', array(), '0.5');
```

PHPStan (2.2.2)

[error] Constant ACORE_URL_PLG not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)